tests/backup/backup_resiliency_test.go

```diff
@@ -1025,3 +1026,252 @@ var _ = Describe("{ScaleMongoDBWhileBackupAndRestore}", func() {
 	})
 })
+
+// RebootNodesWhenBackupsAreInProgress reboots worker nodes from application cluster when backup is in progress
+// source cluster is backup cluster and destination cluster is application cluster for this testcase
+var _ = Describe("{RebootNodesWhenBackupsAreInProgress}", func() {
+	var (
+		contexts          []*scheduler.Context
+		appContexts       []*scheduler.Context
+		appNamespaces     []string
+		cloudCredName     string
+		cloudCredUID      string
+		bkpLocationName   string
+		backupLocationUID string
+		destClusterUid    string
+		srcClusterStatus  api.ClusterInfo_StatusInfo_Status
+		destClusterStatus api.ClusterInfo_StatusInfo_Status
+		backupNames       []string
+		newBackupNames    []string
+		listOfWorkerNodes []node.Node
+	)
+	labelSelectors := make(map[string]string)
+	numberOfBackups, _ := strconv.Atoi(getEnv(maxBackupsToBeCreated, "2"))
+	backupLocationMap := make(map[string]string)
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootNodesWhenBackupsAreInProgress",
+			"Reboots node when backup is in progress", nil, 55817)
+		log.InfoD("Switching cluster context to application[destination] cluster which does not have px-backup deployed")
+		SetDestinationKubeConfig()
+		log.InfoD("Deploying applications required for the testcase on application cluster")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < numberOfBackups; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				appNamespaces = append(appNamespaces, namespace)
+			}
+		}
+		log.Infof("The list of namespaces are %v", appNamespaces)
+	})
+	It("Reboot node when backup is in progress", func() {
+		var sem = make(chan struct{}, numberOfBackups)
+		var wg sync.WaitGroup
+		Step("Validating the deployed applications", func() {
+			log.InfoD("Validating the deployed applications on destination cluster")
+			ValidateApplications(contexts)
+		})
+		Step("Switching cluster context back to source[backup] cluster", func() {
+			log.InfoD("Switching cluster context back to source[backup] cluster")
+			err := SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+		})
+		Step("Adding cloud credential and backup location", func() {
+			log.InfoD("Adding cloud credential and backup location")
+			providers := getProviders()
+			for _, provider := range providers {
+				cloudCredName = fmt.Sprintf("%s-%s-%v", "cloudcred", provider, time.Now().Unix())
+				bkpLocationName = fmt.Sprintf("%s-%s-%v-bl", provider, getGlobalBucketName(provider), time.Now().Unix())
+				cloudCredUID = uuid.New()
+				backupLocationUID = uuid.New()
+				backupLocationMap[backupLocationUID] = bkpLocationName
+				CreateCloudCredential(provider, cloudCredName, cloudCredUID, orgID)
```

Comment:
Please refer to PR #1228. CreateCloudCredential now returns an error.

```
err := CreateCloudCredential(provider, cloudCredName, cloudCredUID, orgID, ctx)
				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential named [%s] for org [%s] with [%s] as provider", cloudCredName, orgID, provider))
```

Reply:
done